### PR TITLE
Update report1.txt

### PR DESCRIPTION
**Pull request by devendravannela**
Update report1.txt

### CODE REVIEW

File: report1.txt [hunk 1]
```diff
@@ -1 +1,6 @@
 This is myt report 1.
+
```
Comment (devopslearning222) [inline report1.txt:2]: Remove blank spaces

File: report1.txt [hunk 2]
```diff
@@ -1 +1,6 @@
...
+This is myt report 1.
+
+v
```
Comment (devopslearning222) [inline report1.txt:5]: Improve the performance of this code

File: report1.txt [hunk 3]
```diff
@@ -1 +1,6 @@
 This is myt report 1.
+
+This is myt report 1.
+
```
Comment (devopslearning222) [inline report1.txt:4]: change the logic

**Review by devopslearning222 — CHANGES_REQUESTED** [review]

File: report1.txt (hunk 1 above)
Comment (devendravannela) [reply report1.txt:2]: done!

File: report1.txt (hunk 2 above)
Comment (devendravannela) [reply report1.txt:5]: This logic having performance in consideration.

**Review by devopslearning222 — APPROVED** [review]

**Pull request merged**